2019-11-30 23:41:10.462 - [Info] MITM Version: 1.26
2019-11-30 23:41:10.468 - [Info] Auto Access Enabled: true
2019-11-30 23:41:10.469 - [Debug] [Init] Auto Access Barrier: true
2019-11-30 23:41:10.491 - [Info] SSH man-in-the-middle server for 172.20.0.4 listening on 0.0.0.0:10103
2019-11-30 23:47:53.632 - [Debug] [Connection] Attacker connected: 221.229.219.188 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-30 23:47:56.183 - [Debug] [Auth] Attacker 221.229.219.188 trying to authenticate with "password"
2019-11-30 23:47:56.186 - [Debug] [Auto Access] Attacker: 221.229.219.188, Threshold: 1, Attempts: 1
2019-11-30 23:47:56.186 - [Debug] [Auto Access] Compromising the honeypot
2019-11-30 23:47:56.187 - [Debug] [Auto Access] Adding the following credentials: "alcanar:alcanar"
2019-11-30 23:47:56.669 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-30 23:47:56.670 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.4
2019-11-30 23:47:56.779 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-30 23:47:57.957 - [Debug] [Connection] Attacker connected: 221.229.219.188 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-30 23:48:00.558 - [Debug] [Auth] Attacker 221.229.219.188 trying to authenticate with "password"
2019-11-30 23:48:00.559 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-30 23:48:00.561 - [Debug] [Auth] Attacker: 221.229.219.188 has so far made 1 attempts. Remaining: 5 attempts
2019-11-30 23:48:01.167 - [Debug] [Connection] Attacker closed the connection
2019-11-30 23:48:01.747 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
2019-11-30 23:58:18.509 - [Debug] [Connection] Client error on ssh server
2019-11-30 23:58:18.510 - [Debug] [Connection] Attacker closed connection
2019-11-30 23:58:18.518 - [Debug] [LXC] Container's OpenSSH server ended connection
2019-11-30 23:58:18.518 - [Debug] [LXC] Container's OpenSSH server closed connection
